core/domain/app_feedback_report_domain_test.py

```diff
@@ -199,6 +240,23 @@ def test_report_web_platform_validation_fails(self) -> None:
             'Domain objects for web reports have not been implemented yet.'):
             self.web_report_obj.validate()
 
+    def test_raise_not_implemented_error_for_web_platform_feedback(
+        self
+        ) -> None:
```

Comment:
Deindent by 4 to match the opening paren. Ditto elsewhere.

Reply:
done.